services/auth/auth-supabase.service.ts

```diff
@@ -66,6 +66,28 @@ class AuthService {
     let { error } = await this.supabase.auth.signOut();
     if (error) return messageHandler.handleError(error.message);
   }
+
+  async resetPassword(email: string): Promise<{} | null> {
+    let { data, error } = await this.supabase.auth.resetPasswordForEmail(
+      email,
+      {
+        redirectTo: `${window.location.origin}/auth/${MODULE.CONFIRM_RESET_PASSWORD}`,
```

Comment:
Why we use origin instead of the ENV variable? Are we pointing to next API? (Havent seen it)

Reply:
We're pointing to the confirm_reset_password page